front/src/modules/ui/input/autosize-text/components/AutosizeTextInput.tsx

```diff
@@ -51,19 +64,59 @@ const StyledBottomRightRoundedIconButton = styled.div`
   height: 0;
   position: relative;
   right: 26px;
-  top: calc(100% - 26.5px);
+  top: 6px;
   width: 0px;
 `;
 
+const StyledSendButton = styled(Button)`
+  margin-left: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledWordCounter = styled.div`
+  color: ${({ theme }) => theme.font.color.light};
+  font-weight: ${({ theme }) => theme.font.weight.medium};
+  line-height: 150%;
+  width: 100%;
+`;
+
+interface StyledBottomContainerProps {
+  isTextAreaHidden: boolean;
+}
+
+const StyledBottomContainer = styled.div<StyledBottomContainerProps>`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin-top: ${({ theme, isTextAreaHidden }) =>
+    isTextAreaHidden ? 0 : theme.spacing(4)};
+`;
+
+const StyledCommentText = styled.div`
+  cursor: text;
+  padding-bottom: ${({ theme }) => theme.spacing(1)};
+  padding-top: ${({ theme }) => theme.spacing(1)};
+`;
+
 export function AutosizeTextInput({
   placeholder,
   onValidate,
   minRows = 1,
+  onFocus,
+  variant = 'icon',
+  buttonTitle,
 }: OwnProps) {
   const [isFocused, setIsFocused] = useState(false);
+  const [isHidden, setIsHidden] = useState(variant === 'button');
```

Comment:
I don't understand this isHidden state and why it would be specific to button variant?

Reply:
Hello @charlesBochet,

In the case of the button variant, the textarea is initially hidden until a user clicks on "Write a comment." This action unhides the textarea and focuses it, making it ready for the user to start typing. However, this behavior differs in the icon variant, where users can immediately start typing without any initial interaction.

https://github.com/twentyhq/twenty/assets/140154534/54ce98d0-1e78-47bc-8f8f-b8010eb51d93